services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -6284,24 +6284,21 @@ private void createControllerClusterIfRequired() {
 
   private void setupStorageClusterAsNeeded(String clusterName) {
     if (!helixAdminClient.isVeniceStorageClusterCreated(clusterName)) {
-      Map<String, String> helixClusterProperties = new HashMap<>();
-      helixClusterProperties.put(ZKHelixManager.ALLOW_PARTICIPANT_AUTO_JOIN, String.valueOf(true));
-      long delayRebalanceTimeMs = multiClusterConfigs.getControllerConfig(clusterName).getDelayToRebalanceMS();
+      ClusterConfig helixClusterConfig = new ClusterConfig(clusterName);
```

Comment:
Helix doesnt have builder for ClusterConfig?

Reply:
Yeah, I didn't find that https://github.com/apache/helix/blob/master/helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java